src/demos/calcite-radio-button.html

```diff
@@ -131,135 +139,187 @@ <h3>Only one checked value (first wins, second two checked)</h3>
+                  </calcite-radio-button>
 
-                    <hr>
+                  <hr>
 
-                    <calcite-radio-button theme="dark" name="firstCheckedWinsDarkDisabledSingleFirstLast" value="1" checked></calcite-radio-button>
-                    <calcite-radio-button theme="dark" name="firstCheckedWinsDarkDisabledSingleFirstLast" value="2" disabled></calcite-radio-button>
-                    <calcite-radio-button theme="dark" name="firstCheckedWinsDarkDisabledSingleFirstLast" value="3" checked></calcite-radio-button>
+                  <calcite-radio-button theme="dark" name="firstCheckedWinsDarkDisabledSingleFirstLast" value="1"
+                    checked></calcite-radio-button>
+                  <calcite-radio-button theme="dark" name="firstCheckedWinsDarkDisabledSingleFirstLast" value="2"
+                    disabled></calcite-radio-button>
+                  <calcite-radio-button theme="dark" name="firstCheckedWinsDarkDisabledSingleFirstLast" value="3"
+                    checked></calcite-radio-button>
 
                   <h3>Only one checked value (first wins, all checked)</h3>
-                    <calcite-radio-button theme="dark" name="firstCheckedWinsDarkAll" value="1" checked></calcite-radio-button>
-                    <calcite-radio-button theme="dark" name="firstCheckedWinsDarkAll" value="2" checked></calcite-radio-button>
-                    <calcite-radio-button theme="dark" name="firstCheckedWinsDarkAll" value="3" checked></calcite-radio-button>
+                  <calcite-radio-button theme="dark" name="firstCheckedWinsDarkAll" value="1" checked>
+                  </calcite-radio-button>
+                  <calcite-radio-button theme="dark" name="firstCheckedWinsDarkAll" value="2" checked>
+                  </calcite-radio-button>
+                  <calcite-radio-button theme="dark" name="firstCheckedWinsDarkAll" value="3" checked>
+                  </calcite-radio-button>
 
-                    <hr>
+                  <hr>
 
-                    <calcite-radio-button theme="dark" name="firstCheckedWinsDarkAllDisabledSingle" value="1" checked></calcite-radio-button>
-                    <calcite-radio-button theme="dark" name="firstCheckedWinsDarkAllDisabledSingle" value="2" checked disabled></calcite-radio-button>
-                    <calcite-radio-button theme="dark" name="firstCheckedWinsDarkAllDisabledSingle" value="3" checked></calcite-radio-button>
+                  <calcite-radio-button theme="dark" name="firstCheckedWinsDarkAllDisabledSingle" value="1" checked>
+                  </calcite-radio-button>
+                  <calcite-radio-button theme="dark" name="firstCheckedWinsDarkAllDisabledSingle" value="2" checked
+                    disabled></calcite-radio-button>
+                  <calcite-radio-button theme="dark" name="firstCheckedWinsDarkAllDisabledSingle" value="3" checked>
+                  </calcite-radio-button>
 
```

Comment:
☝️ changes in this file are purely formatting, I just ran prettier on it to get the HTML to look nicer.